its/src/test/java/org/sonarsource/scanner/ant/it/AntTest.java

```diff
@@ -193,19 +197,14 @@ public void shouldFailIfMissingProperty() {
       .setTargets("sonar");
 
     BuildResult analysisResults = orchestrator.executeBuildQuietly(build);
-    assertThat(analysisResults.getStatus()).isNotEqualTo(0);
+    assertThat(analysisResults.getLastStatus()).isNotEqualTo(0);
 
     String logs = analysisResults.getLogs();
     assertThat(logs).contains("You must define the following mandatory properties", "sonar.projectKey");
   }
 
-  private static boolean containsRule(final String ruleKey, List<Issue> issues) {
-    return Iterables.any(issues, new Predicate<Issue>() {
-      @Override
-      public boolean apply(@Nullable Issue input) {
-        return input != null && ruleKey.equals(input.ruleKey());
-      }
-    });
+  private static boolean containsRule(final String ruleKey, List<Issues.Issue> issues) {
+    return issues.stream().anyMatch(input -> input != null && ruleKey.equals(input.getRule()));
```

Comment:
i would have used a filter for the non-bull thing, then the anymatch. but its more a personal code style, so this is ok too